package.json

```diff
@@ -18,20 +18,30 @@
   ],
   "nativescript": {
     "platforms": {
-      "ios": "1.5.0",
-      "android": "1.5.0"
+      "ios": "3.0.0",
+      "android": "3.0.0"
+    },
+    "tns-ios": {
+      "version": "3.0.0"
     }
   },
-  "author": "Alexander Vakrilov",
+  "author": {
+    "name": "Alexander Vakrilov"
+  },
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/vakrilov/native-script-accelerometer.git/issues"
   },
   "homepage": "https://github.com/vakrilov/native-script-accelerometer",
   "dependencies": {
```

Comment:
Not that we have peerDependency to `tns-core-modules` there is no need for a `dependency`. Please leave only the peer dependency